content/bn/docs/contribute/_index.md

```diff
@@ -0,0 +1,100 @@
+---
+content_type: ধারণা
+title: অবদান রাখো K8s ডকুমেনটেশন
+linktitle: অবদান
+main_menu: true
+no_list: true
+weight: 80
+card:
+  name: অবদান
+  weight: 10
+  title: K8s এ অবদান রাখা শুরু করুন
+---
+
+<!-- overview -->
+
+*Kubernetes নতুন এবং অভিজ্ঞ সকল অবদানকারীদের থেকে উন্নতিকে (immprovements) স্বাগত জানায়!*
+
+{{< note >}}
+সাধারণভাবে Kubernetes অবদান সম্পর্কে আরও জানতে, দেখুন
+[অবদানকারীদের জন্য ডকুমেন্টেশন](https://www.kubernetes.dev/docs/)।
+{{< /note >}}
+
+এই ওয়েবসাইটটি [Kubernetes SIG ডক্স](/docs/contribute/#get-involved-with-sig-docs) দ্বারা রক্ষণাবেক্ষণ করা হয় ।
+
+Kubernetes ডকুমেন্টেশন অবদানকারী:
+
+- বিদ্যমান সামগ্রী উন্নত করুন
+- নতুন কন্টেন্ট তৈরি করুন
+- ডকুমেন্টেশন অনুবাদ করুন
+- Kubernetes রিলিজ চক্রের ডকুমেন্টেশন অংশগুলি পরিচালনা এবং প্রকাশ করুন
+
+<!-- body -->
+
+## শুরু করুন (Get Started)
+
+যে কেউ ডকুমেন্টেশন সম্পর্কে একটি সমস্যা (issue)  খুলতে পারে, বা   পরিবর্তন করতে পারে
+ পুল রিকুয়েস্ট(Pull Request) দেয়ার মাধ্যমে
+[`kubernetes/website` GitHub সংগ্রহস্থল](https://github.com/kubernetes/website) এ। আপনাকে 
+[git](https://git-scm.com/) এবং
+[GitHub](https://lab.github.com/) সম্পর্কে জানতে হবে 
+কুবারনেটস সম্প্রদায়ে কার্যকরভাবে কাজ করার জন্য।
+
+
+ডকুমেন্টেশনের সাথে জড়িত হতে:
+
+1. CNCF [কন্ট্রিবিউটর লাইসেন্স চুক্তি](https://github.com/kubernetes/community/blob/master/CLA.md) স্বাক্ষর করুন।
+1. [ডকুমেন্টেশন রিপোজিটরি](https://github.com/kubernetes/website) এবং ওয়েবসাইটের [স্ট্যাটিক সাইট জেনারেটর](https://gohugo.io) এর সাথে নিজেকে পরিচিত করুন । 
+1. নিশ্চিত করুন যে আপনি প্রাথমিক প্রক্রিয়াগুলি বুঝতে পেরেছেন৷
+   [একটি পুল অনুরোধ খোলার](/docs/contribute/new-content/open-a-pr/) এবং
+   [পরিবর্তন পর্যালোচনা করার](/docs/contribute/review/reviewing-prs/)।
+   
```

Comment:
There should be a mermaid diagram here (refer [English version](https://github.com/kubernetes/website/blob/main/content/en/docs/contribute/_index.md?plain=1#L55)). We should avoid omitting anything as much as we can

Reply:
@anubha-v-ardhan Updated all of the contents